network/endpoint.go

```diff
@@ -144,6 +150,18 @@ func (ep *endpoint) getInfo() *EndpointInfo {
 		Id:          ep.Id,
 		IPAddresses: ep.IPAddresses,
 		Data:        make(map[string]interface{}),
+		MacAddress:  ep.MacAddress,
+		SandboxKey:  ep.SandboxKey,
+		IfIndex:     0, // Azure CNI supports only one interface
```

Comment:
why this value is hardcoded.? Whicn interface it points to?

Reply:
hardcoded this to 0 with Sushant. Let me touch base with him again if this needs to be a declared const.